.github/workflows/ci-mac-os.yml

```diff
@@ -37,12 +37,13 @@ jobs:
         python -m pip install --upgrade pip
         
         # install xml libraries and requirements for lateXML
-        brew install cpanm
-        brew install libxml2 libxslt
-        cpanm XML::LibXML XML::XSLT
-        cpanm XML::LibXSLT --force
-        cpanm Archive::Zip DB_File File::Which Getopt::Long Image::Size IO::String JSON::XS LWP MIME::Base64 Parse::RecDescent Pod::Parser Text::Unidecode Test::More URI UUID::Tiny
-
+        # brew install cpanm
+        brew install libxml2     
+        brew install --force libxslt
+        
+        # cpanm -v --notest Archive::Zip DB_File File::Which Getopt::Long Image::Size IO::String JSON::XS LWP MIME::Base64 Parse::RecDescent Pod::Parser Text::Unidecode Test::More URI XML::LibXML XML::LibXSLT UUID::Tiny
```

Comment:
Could you add that this list comes from lateXML docker file LaTeXML/tools/ci/Dockerfile